docs/guides/blocks-block-from-components.md

```diff
@@ -202,9 +202,9 @@ export const Card = (props) => {
 };
 ```
 
-Here we'll use our ready made component (built out of other components - reusability), and wrap it in [React Fragment](https://reactjs.org/docs/fragments.html). It is a pattern used so that we can return multiple elements.  
+Here we'll use our ready-made component (built out of other components - reusability), and wrap it in [React Fragment](https://reactjs.org/docs/fragments.html). It is a pattern used so that we can return multiple elements.  
```

Comment:
`It is a pattern used to return multiple elements.`

Reply:
👍